### PR TITLE
Add alumni section to team.html

### LINKED ISSUE #739: Missing alumni section for past SEF members
**Is your feature request related to a problem?**
Currently there is no section for past SEF members.

**Describe the solution you'd like**
We can add a new "alumni" section in team.html and add past SEF members in it.

### PR DESCRIPTION
**Pull request by Gimhan-minion**
## Purpose

The purpose of this PR is to fix #739 

## Goals
To add past SEF members to the SEF website.

## Approach
1. Created a new section in team.json file and add all the past members in it.
2. Uploaded each of their images.
3. Made changes in team.html as required.
  
### Preview Link

https://pr-740-sef-site.surge.sh/team.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by anjula-sack — COMMENTED** [review]
Are we going to add LinkedIn links? If we don’t we can remove it? @Gimhan-minion

**Review by anjula-sack — DISMISSED** [review]
Great Job! @Gimhan-minion

File: src/main/webapp/team.html [hunk 1]
```diff
@@ -131,6 +131,19 @@
...
+<section class="section pb-0">
+    <div class="container">
+        <p class="display-3 text-center mb-5 ">Alumni</p>
+
+        <!--Profile cards goes here-->
+        <div id="alumni"></div>
+
+        <div class="row justify-content-center">
+            <button id="btnShowMore_alumni" class="btn btn-link btn-outline-white btn-info">SEE MORE</button>
+        </div>
```
Comment (Gravewalker666) [inline src/main/webapp/team.html]: Add a margin here, the same thing as the previous PR.

image

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: src/main/webapp/team.html (hunk 1 above)
Comment (Gimhan-minion) [reply src/main/webapp/team.html]: Done

**Review by Piumal1999 — APPROVED** [review]
Good job @Gimhan-minion Thank you

**Review by Gravewalker666 — APPROVED** [review]

**Pull request merged**